### PR TITLE
ECO 5613

### PR DESCRIPTION
**Pull request by Raulkg**
Implemented Font Swap Display Strategy for custom fonts - Improves performance - Lighthouse Metric

### CODE REVIEW

File: web/less/room.less [hunk 1]
```diff
@@ -223,10 +223,9 @@ body {
       display: none;
     }
 
-     @media @iphoneAll @Landscape {
-             width: 2px;
-             min-width: 2px;
-     }
+    @media @iphoneAll @Landscape {
+      display: none;
+    }
```
Comment (jeffswartz) [inline web/less/room.less:228]: I don't understand this change.

**Review by jeffswartz — CHANGES_REQUESTED** [review]
Please resolve the merge conflicts.

File: web/less/room.less (hunk 1 above)
Comment (Raulkg) [reply web/less/room.less:228]: this is to hide the audio switch  in landscape mode which was also hiding in portrait mode

Comment (jeffswartz) [reply web/less/room.less]: Why do we set `display: none;` for `@media @smartphonesPortrait` and `@media @iphoneAll @Landscape` as opposed to, say, `@media @smartphonesPortrait` and `@media @smartphonesLandscape`? Why do we have separate variables for `@smartphones` and `@iphone`?

**Review by jeffswartz — APPROVED** [review]

**Pull request merged**